Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1167,31 +1167,6 @@ class BrowserViewController: UIViewController {
         // Remember whether or not a desktop site was requested
         tab.desktopSite = webView.customUserAgent?.isEmpty == false
     }
-    
-    // MARK: open in helper utils
-    func addViewForOpenInHelper(_ openInHelper: OpenInHelper?) {
```

Comment:
removing all of this is awesome. But we can go even further! I dont think `webViewContainerToolbar` is used at all anymore.

Reply:
There are also hacks in TabScrollController that could be cleaned up. Like the PDF check we do